app/assets/javascripts/discourse/app/components/ignored-user-list.js

```diff
@@ -11,7 +11,7 @@ export default Component.extend({
       this.items.removeObject(item);
       User.findByUsername(item).then((user) => {
         user
-          .updateNotificationLevel("normal")
+          .updateNotificationLevel("normal", null, this.model)
```

Comment:
Yeah it's tricky, I don't like this null in the middle either. Will make a change to named params, hopefully it's an easy cut-over and no plugins are using this, otherwise I'll need to do a deprecation dance.

Reply:
Thankfully nothing else was using this, so no need for contortions